lib/shp-write-stream/index.js

```diff
@@ -4,7 +4,7 @@ const pump = require('pump');
 const shpWritter = require('./shp');
 const DbfWritter = require('./dbf');
 class Input extends Writable {
-  constructor(schema, createStream, opts, cb) {
+  constructor(schema, createStream, opts, cb, enc) {
```

Comment:
You'd better reuse the `opts` optional object with an `encoding` key.

Reply:
yes, right.